### PR TITLE
PMMI Nav, Layout updates

### PR DESCRIPTION
**Pull request by brandonbk**

### CODE REVIEW

File: sites/oemmagazine/server/templates/website-section/index.marko [hunk 1]
```diff
@@ -31,29 +26,16 @@ $ const { id, alias, name, pageNode } = data;
...
-    <marko-web-resolve-page|{ data: section }| node=pageNode>
-      $ const aliases = hierarchyAliases(section);
-      $ const nativeX = { index: 1, name: "load-more" };
-      <marko-web-query|{ nodes }|
-        name="website-scheduled-content"
-        params={ sectionId: id, limit: 8, skip: 5, queryFragment }
-      >
-        <website-content-card-deck-flow nodes=nodes aliases=aliases native-x=nativeX ad-index=2 ad-name="imu1" />
-      </marko-web-query>
-    </marko-web-resolve-page>
```
Comment (solocommand) [inline sites/oemmagazine/server/templates/website-section/index.marko]: You can't remove this query, or page views will fire twice.

File: sites/packworld/server/templates/website-section/index.marko [hunk 2]
```diff
@@ -12,49 +12,30 @@ $ const { id, alias, name, pageNode } = data;
...
-      $ const aliases = hierarchyAliases(section);
-      $ const nativeX = { index: 1, name: "load-more" };
-      <marko-web-query|{ nodes }|
-        name="website-scheduled-content"
-        params={ sectionId: id, limit: 8, skip: 5, queryFragment }
-      >
-        <website-content-card-deck-flow nodes=nodes aliases=aliases native-x=nativeX ad-index=2 ad-name="imu1" />
-      </marko-web-query>
-    </marko-web-resolve-page>
-
```
Comment (solocommand) [inline sites/packworld/server/templates/website-section/index.marko]: Can't remove, pageviews.

File: sites/profoodworld/server/templates/website-section/index.marko [hunk 3]
```diff
@@ -12,48 +12,30 @@ $ const { id, alias, name, pageNode } = data;
...
-      $ const aliases = hierarchyAliases(section);
-      $ const nativeX = { index: 1, name: "load-more" };
-      <marko-web-query|{ nodes }|
-        name="website-scheduled-content"
-        params={ sectionId: id, limit: 8, skip: 5, queryFragment }
-      >
-        <website-content-card-deck-flow nodes=nodes aliases=aliases native-x=nativeX ad-index=2 ad-name="imu1" />
-      </marko-web-query>
-    </marko-web-resolve-page>
-
```
Comment (solocommand) [inline sites/profoodworld/server/templates/website-section/index.marko]: Can't remove, pageviews.

File: sites/automationworld/server/templates/dynamic-page/index.marko [hunk 4]
```diff
@@ -10,7 +10,6 @@ $ const { id, alias, pageNode } = data;
   </@head>
   <@page>
     <marko-web-gam-define-display-ad ...GAM.getAdUnit({ name: "leaderboard" }) modifiers=["top-of-page"] />
-    <marko-web-gam-define-display-ad ...GAM.getAdUnit({ name: "article-top-below-head" }) modifiers=["top-of-page"] />
```
[the remark this answers is not in the document]
Comment (brandonbk) [reply sites/automationworld/server/templates/dynamic-page/index.marko]: Leaving as-is until we receive clarification on which page(s) it needs to be included.

**Pull request merged**